lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -43,6 +49,20 @@ abstract class SqlParserTestBase : TestBase() {
 
     protected fun parse(source: String): PartiqlAst.Statement = parser.parseAstStatement(source)
 
+    private fun assertSexpEquals(
```

Comment:
This comes from `TestBase`... since this class was the only place it was being used, I moved it here.